aten/src/ATen/native/cudnn/LossCTC.cpp

```diff
@@ -94,7 +94,7 @@ bool _use_cudnn_ctc_loss(
       // target length < 256 is documented, but we see illegal memory accesses
       // when target lengths > input lengths for CuDNN
       use_cudnn &=
```

Comment:
`use_cudnn &&=`?

Reply:
`&&=` is not a C++ operator https://en.cppreference.com/w/cpp/language/operator_precedence

Reply:
Right.
Then `use_cudnn = use_cudnn && ...`?